xslt/Plane.xsl

```diff
@@ -76,12 +77,26 @@
             </xsl:call-template>
             <xsl:apply-templates select="METADATA/Tags/DetectorState/CameraState/ExposureTime"/>
             <xsl:apply-templates select="METADATA" />
-
+            <!-- 
+                The element below is a reference to another element that contains the Acquisition Time
+                for this subblock. This is neccessary because the Plane element doesn't have a way to
+                store this information directly according to the OME XSD.
+            -->
+            <xsl:element name="ome:AnnotationRef">
+                <xsl:attribute name="ID">
+                    <xsl:text>Annotation:SubblockAcquisition</xsl:text>
+                    <!-- Add the name and value of each attribute to the ID of this element -->
+                    <xsl:for-each select="@*">
+                        <xsl:value-of select="name(.)"/>
+                        <xsl:value-of select="."/>
+                    </xsl:for-each>
+                </xsl:attribute>
+            </xsl:element>
         </xsl:element>
     </xsl:template>
 
 
-    <xsl:template match="Subblocks">
+    <xsl:template match="Subblocks" mode="plane">
```

Comment:
Nice use of modes to match the subblocks multiple times, I assume to populate the different parts of the output xml (plane and structured annotations).